src/Nautilus/Asset/AssetBuilder.ts

```diff
@@ -89,6 +89,36 @@ export class AssetBuilder implements IAssetBuilder {
     return this
   }
 
+  setNftTokenName(name: string) {
+    this.asset.nftCreateData.name = name
+
+    return this
+  }
+
+  setNftTokenSymbol(symbol: string) {
+    this.asset.nftCreateData.symbol = symbol
+
+    return this
+  }
+
+  setNftTokenUri(uri: string) {
+    this.asset.nftCreateData.tokenURI = uri
+
+    return this
+  }
+
+  setNftTokenTransferable(transferable = true) {
```

Comment:
thought: I think I remember it is a compliance requirement to be non-transferable. We should make the default false if we decide to expose this setting.

Reply:
the default is false. If this function is not called. For calling this function the default should be `true` in my opinion as it makes sense reading the code that way (DX)